ep/build-velox/src/setup-centos7.sh

```diff
@@ -163,19 +163,17 @@ function install_boost {
 
 function install_protobuf {
   cd "${DEPENDENCY_DIR}"
-  wget https://github.com/protocolbuffers/protobuf/releases/download/v21.4/protobuf-all-21.4.tar.gz
-  tar -xzf protobuf-all-21.4.tar.gz
-  cd protobuf-21.4
+  wget_and_untar https://github.com/protocolbuffers/protobuf/releases/download/v21.4/protobuf-all-21.4.tar.gz protobuf
+  cd protobuf
```

Comment:
Use ` pushd protobuf`, then add `popd` at the end of this function. Then, we can avoid wrong directory may be possibly used when executing some other functions. See https://github.com/apache/incubator-gluten/pull/9294.

Reply:
👍